tool/tsh/proxy_test.go

```diff
@@ -166,6 +174,57 @@ func testLeafClusterSSHAccess(t *testing.T, s *suite) {
 	require.NoError(t, err)
 }
 
+func testJumpHostSSHAccess(t *testing.T, s *suite) {
```

Comment:
I think converting this test cases to a test table would make this test easier to understand.